src/test/kotlin/it/pagopa/wallet/eventdispatcher/warmup/utils/WarmupRequestsTest.kt

```diff
@@ -0,0 +1,60 @@
+package it.pagopa.wallet.eventdispatcher.warmup.utils
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import it.pagopa.wallet.eventdispatcher.common.cdc.WalletDeletedEvent
+import it.pagopa.wallet.eventdispatcher.configuration.CdcSerializationConfiguration
+import it.pagopa.wallet.eventdispatcher.configuration.SerializationConfiguration
+import it.pagopa.wallet.eventdispatcher.domain.WalletCreatedEvent
+import it.pagopa.wallet.eventdispatcher.queues.WalletCdcQueueConsumer
+import it.pagopa.wallet.eventdispatcher.queues.WalletExpirationQueueConsumer
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import org.mockito.Mockito.mock
+import org.springframework.test.context.TestPropertySource
+import reactor.test.StepVerifier
+
+@TestPropertySource(locations = ["classpath:application.test.properties"])
+class WarmupRequestsTest {
+
+    @Test
+    fun `Should handle WalletCreatedEvent`() {
+        val serializationConfiguration = SerializationConfiguration()
+        val objectMapper: ObjectMapper = serializationConfiguration.objectMapperBuilder().build()
+        val azureJsonSerializer = serializationConfiguration.azureJsonSerializer(objectMapper)
+
+        val walletExpirationQueueConsumer =
+            WalletExpirationQueueConsumer(
+                walletsApi = mock(),
+                azureJsonSerializer = azureJsonSerializer,
+                tracing = mock()
+            )
+        val payload = WarmupRequests.getWalletCreatedEvent()
+        val result = walletExpirationQueueConsumer.parseEvent(payload)
+
+        StepVerifier.create(result).assertNext { event ->
+            assertEquals(WalletCreatedEvent::class.java, event::class.java)
+        }
```

Comment:
```suggestion
        StepVerifier
            .create(result)
            .assertNext { queueEvent ->
                assertTrue(
                    queueEvent.data is WalletCreatedEvent
                )
            }
            .verifyComplete()
```

watch out, without calling verifyComplete on StepVerifier object reactive code is not actually executed and test not performed